specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.2-preview.1/TextAnalytics.json

```diff
@@ -846,7 +850,7 @@
               "type": "string"
             },
             "loggingOptOut": {
-              "default": true,
+              "default": false,
```

Comment:
sorry to bug you on this again, but was this change intentional for `PiiTask`? I thought we wanted to opt out of logging specifically for PII and healthcare, but correct me if I'm wrong about this.

Reply:
Oh ok, yes, this was probably unintentional.  I might have been looking at the wrong line of code when I did this.  Good to know that logging opt out is intentionally different for those two services.  I'll double check things and check in a fix soon.

Reply:
Reverted this change.  Let me know if you notice any other issues.  :)